Chaap/Chaap/Common/Components/CHBlurView.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  BlurEffect.swift
+//  Chaap
+//
+//  Created by Enoch on 7/18/25.
+//
+
+import Foundation
+import SwiftUI
+
+struct CHBlurView: UIViewRepresentable {
+    var style: UIBlurEffect.Style
+    
+    func makeUIView(context: Context) -> some UIVisualEffectView {
+        let view = UIVisualEffectView(effect: UIBlurEffect(style: style))
+        return view
+    }
+    
+    func updateUIView(_ uiView: UIViewType, context: Context) { }
+}
+
+
+// 사용방법
+// .background(BlurView(style: .systemUltraThinMaterial))
```

Comment:
파일명 변경에 따라 사용방법 예시도 변경
```suggestion
// .background(CHBlurView(style: .systemUltraThinMaterial))
```